client/tokenized-express-checkout/blocks/components/express-checkout-button-preview.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useMemo, useRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getExpressCheckoutButtonAppearance } from 'wcpay/express-checkout/utils';
```

Comment:
Since this file is within the `tokenized-express-checkout`, it's best to include the functions from that directory, rather than the `express-checkout` directory (which doesn't include "tokenized" cart changes).

Otherwise the bundles are going to get mixed, and there might be modifications in the "tokenized" functions that wouldn't be considered.

```suggestion
import { getExpressCheckoutButtonAppearance } from '../../utils';
```